bundle/internal/schema/main.go

```diff
@@ -0,0 +1,93 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"reflect"
+
+	"github.com/databricks/cli/bundle/config"
+	"github.com/databricks/cli/bundle/config/variable"
+	"github.com/databricks/cli/libs/jsonschema"
+)
+
+func interpolationPattern(s string) string {
+	return fmt.Sprintf(`\$\{(%s(\.[a-zA-Z]+([-_]?[a-zA-Z0-9]+)*(\[[0-9]+\])*)*(\[[0-9]+\])*)\}`, s)
```

Comment:
It seems like the whole `\.[a-zA-Z]+([-_]?[a-zA-Z0-9]+)*(\[[0-9]+\])*)*(\[[0-9]+\])` section should be + (more than 1) rather than *

Reply:
Why do you need `(\[[0-9]+\])*` ? isn't it already covered by `[a-zA-Z0-9]+` before?

Reply:
Thanks! Yeah that makes sense. I initially just copied the regex we used for variable interpolation but your suggestions make them better here.